src/game/client/components/menus.h

```diff
@@ -628,21 +605,35 @@ class CMenus : public CComponent
 		NUM_BROWSER_COLS,
 	};
 
-	struct CColumn
+	int m_SidebarTab;
+	bool m_SidebarActive;
+	bool m_ShowServerDetails;
+	int m_LastBrowserType;
+	int m_aSelectedFilters[IServerBrowser::NUM_TYPES];
+	char m_LastServerAddress[sizeof(g_Config.m_UiServerAddress)];
+	static CColumn ms_aBrowserCols[NUM_BROWSER_COLS];
+
+	CBrowserFilter* GetSelectedBrowserFilter()
 	{
-		int m_ID;
-		int m_Sort;
-		CLocConstString m_Caption;
-		int m_Direction;
-		float m_Width;
-		int m_Flags;
-		CUIRect m_Rect;
-		CUIRect m_Spacer;
-		CUI::EAlignment m_Align;
-	};
+		const int Tab = ServerBrowser()->GetType();
+		if(m_aSelectedFilters[Tab] == -1)
+			return NULL;
```

Comment:
I believe you are supposed to use `0` or `0x0` in Teeworlds instead of `NULL`. The rationale being that `NULL` is sometimes not zero.

Reply:
Fixed it. That documentation file is also missing the prefix `_pfn` for function pointers.